src/Tools/dotnet-dsrouter/dotnet-dsrouter.csproj

```
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework Condition="'$(BundleTools)' == 'true'">netcoreapp3.1</TargetFramework>
+    <TargetFramework Condition="'$(BundleTools)' != 'true'">netcoreapp2.1</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ToolCommandName>dotnet-dsrouter</ToolCommandName>
+    <RootNamespace>Microsoft.Diagnostics.Tools.DSRouter</RootNamespace>
```

In general, we try to avoid abbreviations in namespaces. I would also change the tool name to be spelled out as well, but brevity is also appreciated at the command line, so it's fine as is 😄 
```suggestion
    <RootNamespace>Microsoft.Diagnostics.Tools.DiagnosticServerRouter</RootNamespace>
```

---

I can change the namespace, but as you said, writing `dotnet-dsrouter.exe` will probably be much more convenient than `dotnet-diagnosticsserverrouter.exe`.